src/bin/vip-validate-env.js

```diff
@@ -0,0 +1,362 @@
+	const resultCounter = harmonia.countResults( true );
+
+	const testSuiteResults = results.filter( result => result instanceof TestSuiteResult );
+
+	// Print the results
+	logToConsole( '\n' + chalk.bgGray( '        HARMONIA RESULTS        \n' ) );
+	if ( resultCounter[ TestResultType.Skipped ] ) {
+		logToConsole( ` ${ chalk.bold.bgGrey( ' SKIPPED ' ) } - ${ chalk.bold( resultCounter[ TestResultType.Skipped ] ) } tests` );
+	}
+	if ( resultCounter[ TestResultType.Success ] ) {
+		logToConsole( ` ${ chalk.bold.bgGreen( ' PASSED ' ) } - ${ chalk.bold( resultCounter[ TestResultType.Success ] ) } tests` );
+	}
+	if ( resultCounter[ TestResultType.PartialSuccess ] ) {
+		logToConsole( ` ${ chalk.bold.bgYellow( ' PARTIAL SUCCESS ' ) } - ${ chalk.bold( resultCounter[ TestResultType.PartialSuccess ] ) } tests` );
+	}
+	if ( resultCounter[ TestResultType.Failed ] ) {
+		logToConsole( ` ${ chalk.bold.bgRed( ' FAILED ' ) } - ${ chalk.bold( resultCounter[ TestResultType.Failed ] ) } tests` );
+	}
+	if ( resultCounter[ TestResultType.Aborted ] ) {
+		logToConsole( ` ${ chalk.bold.bgRedBright( ' ABORTED ' ) } - ${ chalk.bold( resultCounter[ TestResultType.Aborted ] ) } tests` );
+	}
+
+	logToConsole();
+	logToConsole( ` > Total of ${ chalk.bold( results.length ) } tests executed, ${ testSuiteResults.length } of which are Test Suites.` );
+	logToConsole();
+	// If there is a Aborted test result
+	if ( resultCounter[ TestResultType.Aborted ] ) {
+		logToConsole( `${ chalk.bold.bgRedBright( '  NOT PASS  ' ) } There was a critical failure that makes the application ` +
+			'incompatible with VIP Go. Please review the results and re-run the tests.' );
+		process.exit( 1 );
+	}
+
+	// If there is only a partial success, but no failures
+	if ( resultCounter[ TestResultType.PartialSuccess ] && ! resultCounter[ TestResultType.Failed ] ) {
+		logToConsole( `${ chalk.bold.bgYellow( '  PASS  ' ) } The application has passed the required tests, but it does not follow all the recommendations.` );
+		logToConsole( 'Please review the results.' );
+		process.exit( 0 );
+	}
+
+	// If there is a failure
+	if ( resultCounter[ TestResultType.Failed ] ) {
+		logToConsole( `${ chalk.bold.bgRed( '  NOT PASS  ' ) } The application has failed some tests, and will very likely have problems in a production ` +
+			'environment. Please review all the errors found in the results.' );
+		process.exit( 0 );
+	}
+
+	logToConsole( `${ chalk.bold.bgGreen( '  PASS  ' ) } Congratulations. The application passes all the tests.` );
+	process.exit( 0 );
+}
+
+function validateArgs( opt ): {} {
+	const args = {};
+
+	// Verbose
+	if ( opt.verbose ) {
+		Harmonia.setVerbosity( true );
+	}
+
+	// Set path
+	if ( opt.path ) {
+		Harmonia.setCwd( opt.path );
+	}
+
+	// If the JSON option is enabled, all the stdout should be suppressed to prevent polluting the output.
+	if ( opt.json ) {
+		suppressOutput = true;
+		outputJson = true;
+	}
+
+	if ( opt.output ) {
+		outputFile = opt.output;
+	}
```

Comment:
I like the idea, but since it's a CLI I think we can leave this out and let the user redirect (`>`, `|`) output as needed.